### PR TITLE
www.exoclick.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/121436

<details><summary>Screenshot:</summary>

Снимок экрана 2022-06-14 в 14 49 54

</details><br/>

### CODE REVIEW

**Review by zloyden — COMMENTED** [review]
Let's wait for a response from `Easylist`.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/whitelist.txt
Comment (zloyden) [inline EnglishFilter/sections/whitelist.txt:401]: As I can see, we also have the rule in exclusions. It seems it's obsolete now.
I'm not sure that we have to unblock it due to https://forums.lanik.us/viewtopic.php?p=158751-exoclick-com.
@Alex-302 the last word is yours.

Comment (Alex-302) [reply EnglishFilter/sections/whitelist.txt:401]: Answered to user.

**Pull request closed without merging**
